sklearn/metrics/_regression.py

```diff
@@ -1249,13 +1251,15 @@ def max_error(y_true, y_pred):
 
 def _mean_tweedie_deviance(y_true, y_pred, sample_weight, power):
     """Mean Tweedie deviance regression loss."""
+    xp, _ = get_namespace(y_true, y_pred)
     p = power
+    pow_f = np.power if _is_numpy_namespace(xp) else xp.pow
```

Comment:
Could you define `def pow` for `sklearn.utils._array_api._NumPyAPIWrapper` instead (it will be the type of `xp` for the numpy namespace, this is how we manage compatibility with array api primitives that are different in numpy)